js-api-doc/legacy/function.d.ts

```diff
@@ -1,94 +1,661 @@
+     *
+     * // color is `#b37399`.
+     * color.getR(); // 179
+     * ```
+     */
     getR(): number;
+
+    /**
+     * Sets the red channel of the color. The value must be an integer between 0
+     * and 255 (inclusive).
+     *
+     * @deprecated Use [[constructor]] instead.
+     */
     setR(value: number): void;
+
+    /**
+     * Returns the green channel of the color as an integer from 0 to 255.
+     *
+     * @example
+     *
+     * ```js
+     * // color is `#6b717f`.
+     * color.getG(); // 113
+     *
+     * // color is `#b37399`.
+     * color.getG(); // 115
+     * ```
+     */
     getG(): number;
+
+    /**
+     * Sets the green channel of the color. The value must be an integer between
+     * 0 and 255 (inclusive).
+     *
+     * @deprecated Use [[constructor]] instead.
+     */
     setG(value: number): void;
+
+    /**
+     * Returns the blue channel of the color as an integer from 0 to 255.
+     *
+     * @example
+     *
+     * ```js
+     * // color is `#6b717f`.
+     * color.getB(); // 127
+     *
+     * // color is `#b37399`.
+     * color.getB(); // 153
+     * ```
+     */
     getB(): number;
+
+    /**
+     * Sets the blue channel of the color. The value must be an integer between
+     * 0 and 255 (inclusive).
+     *
+     * @deprecated Use [[constructor]] instead.
+     */
     setB(value: number): void;
+
+    /**
+     * Returns the alpha channel of the color as a number from 0 to 1.
+     *
+     * @example
+     *
+     * ```js
+     * // color is `#6b717f`.
+     * color.getA(); // 1
+     *
+     * // color is `transparent`.
+     * color.getA(); // 0
+     * ```
+     */
     getA(): number;
+
+    /**
+     * Sets the alpha channel of the color. The value must be between 0 and 1
+     * (inclusive).
+     *
+     * @deprecated Use [[constructor]] instead.
+     */
     setA(value: number): void;
   }
 
+  /**
+   * Sass's [list type](https://sass-lang.com/documentation/values/lists).
+   *
+   * <div class="sl-c-callout sl-c-callout--warning"><h3>⚠ Heads up!</h3>
+   *
+   * This list type’s methods use 0-based indexing, even though within Sass
+   * lists use 1-based indexing. These methods also don’t support using negative
+   * numbers to index backwards from the end of the list.
+   *
+   * </div>
+   */
   export class List {
+    /**
+     * Creates a new Sass list.
+     *
+     * <div class="sl-c-callout sl-c-callout--warning"><h3>⚠ Heads up!</h3>
+     *
+     * The initial values of the list elements are undefined. These elements
+     * must be set using [[setValue]] before accessing them or passing the list
+     * back to Sass.
+     *
+     * </div>
+     *
+     * @example
+     *
+     * ```js
+     * var list = new sass.types.List(3);
+     * list.setValue(0, new sass.types.Number(10, "px"));
+     * list.setValue(1, new sass.types.Number(15, "px"));
+     * list.setValue(2, new sass.types.Number(32, "px"));
+     * list; // 10px, 15px, 32px
```

Comment:
Nit (here and below, and same for maps):

```suggestion
     * console.log(list); // 10px, 15px, 32px
```
or
```suggestion
     * list.toString(); // 10px, 15px, 32px
```

Reply:
I think this one's all right—the comment indicates the Sass value, which isn't necessarily the result users will get from calling `.toString()`, it's just the notional value associated with the object.